modules/products/bitbucket/nfs/nfs-server/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: nfs-server
+description: A NFS server for testing purposes. Do not use in production.
+
+type: application
+version: 0.1.0
```

Comment:
- [ ] bump this version to `0.2.0` as it is different to the one that we have in repository